packages/server/test/unit/modes/run_spec.coffee

```diff
@@ -435,6 +464,7 @@ describe "lib/modes/run", ->
         name: "foo.mp4"
         cname: "foo-compressed.mp4"
         videoCompression: 32
+        videoUpload: true
```

Comment:
is this necessary? I figured the default value would be used, which is also true?